packages/vite/src/node/config.ts

```diff
@@ -323,6 +323,7 @@ export type ResolvedConfig = Readonly<
     inlineConfig: InlineConfig
     root: string
     base: string
+    rawBase: string
```

Comment:
Let's mark `rawBase` as internal. I would prefer to have this option as you did here, but there was a push in the last team meeting to go there only when there is a real use case out of the Vite core middleware.